candi/bashible/detect_bundle.sh

```diff
@@ -26,10 +26,15 @@ if [ -e /etc/os-release ]; then
       echo "ubuntu-lts"
       exit 0
     ;;
-    debian-9|debian-10|debian-11|astra-2.12.*)
+    debian-9|debian-10|debian-11)
       echo "debian"
       exit 0
     ;;
+    # Works only in EE/FE edition
```

Comment:
I think we can copy-paste the file to the ee/fe folder to prevent unexpected user errors.

Reply:
yes, we copy paste file